beets/library.py

```diff
@@ -1467,8 +1467,10 @@ def tmpl_aunique(self, keys=None, disam=None):
 
         keys = keys or 'albumartist album'
         disam = disam or 'albumtype year label catalognum albumdisambig'
+        bracket = bracket or '[]'
         keys = keys.split()
         disam = disam.split()
+        bracket = None if bracket is ' ' else list(bracket)
```

Comment:
- Is there a reason the condition is `bracket is ' '` instead of using the empty string `''`?
- `is` might work sometimes for comparing strings, but `==` is the right operator to use here.

```py
>>> 'a' is 'a'
True
>>> 'a' * 100 is 'a' * 100
False
>>> 'a' * 100 == 'a' * 100
True
```